arch/arm/src/cxd56xx/cxd56_udmac.c

```diff
@@ -67,8 +68,8 @@ struct dma_channel_s
 
 struct dma_controller_s
 {
-  sem_t exclsem; /* Protects channel table */
-  sem_t chansem; /* Count of free channels */
+  mutex_t lock;     /* Protects channel table */
+  sem_t chansem;    /* Count of free channels */
```

Comment:
```suggestion
  mutex_t lock;                 /* Protects channel table */
  sem_t chansem;                /* Count of free channels */
```
just to align with `dma_channel_s` above